### PR TITLE
Use the master branch instead of stacklight

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

**Comment by simonpasquier** [comment]
**Do not merge until all stacklight branches are merged into master.**

File: classes/cluster/mk22_full_scale/fuel/config.yml [hunk 1]
```diff
@@ -43,33 +43,25 @@ parameters:
...
-            haproxy:
-              revision: stacklight
-            heat:
-              revision: stacklight
+            grafana:
+              state:
+                grafana3_datasource.py:
+                  enabled: true
+                grafana3_dashboard.py:
+                  enabled: true
```
Comment (elemoine) [inline classes/cluster/mk22_full_scale/fuel/config.yml]: It looks like this is not necessary anymore, no? See https://github.com/Mirantis/mk-lab-salt-model/pull/144/files.

**Comment by dkalashnik** [comment]
Guys, we need this ASAP

**Comment by simonpasquier** [comment]
@elemoine can you review it again?

**Pull request merged**